docs/user-guide/remapping.ipynb

```diff
@@ -0,0 +1,557 @@
+{
```

Comment:
<p>Maybe mention somewhere that the Nearest Neighbor method is a distance-based approach, using an internal KD or Ball Tree to determine the distances? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/807/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>